Example/ViewController.m

```diff
@@ -16,38 +18,23 @@ @interface ViewController ()
 @property (nonatomic, strong) DMEPullClient *dmeClient;
 @property (nonatomic, strong) LogViewController *logVC;
 @property (nonatomic, strong) DMEPullConfiguration *configuration;
+@property (nonatomic, strong) DMEOAuthObject *accessToken;
 
 @end
 
 @implementation ViewController
 
-- (void)viewDidLoad {
+- (void)viewDidLoad
+{
     [super viewDidLoad];
     
-    // - GET STARTED -
-    
-    // - REPLACE 'YOUR_APP_ID' with your App ID. Also don't forget to set the app id in CFBundleURLSchemes.
-    NSString *appId = @"YOUR_APP_ID";
-    
-    // - REPLACE 'YOUR_CONTRACT_ID' with your contract ID.
-    NSString *contractId = @"YOUR_CONTRACT_ID";
-    
-    // - REPLACE 'YOUR_P12_FILE_NAME' with .p12 file name (without the .p12 extension) provided by digi.me Ltd.
-    NSString *p12Filename = @"YOUR_P12_FILE_NAME";
-    
-    // - REPLACE 'YOUR_P12_PASSWORD' with password provided by digi.me Ltd.
-    NSString *p12Password = @"YOUR_P12_PASSWORD";
-    
-    self.configuration = [[DMEPullConfiguration alloc] initWithAppId:appId contractId:contractId p12FileName:p12Filename p12Password:p12Password];
-    self.configuration.debugLogEnabled = YES;
```

Comment:
I understand why this section was taken out (for testing purposes) but we need to add it back in! Remember, this is the entry point for most developers trying out SDK for the first time.